README.md

```diff
@@ -37,7 +37,7 @@ which provides access to the [Builder API](https://github.com/clap-rs/clap/blob/
 
 <!-- Copied from examples/demo.{rs,md} -->
 ```rust,no_run
-use clap::Parser;
+use clap::Parser; // Requires; features = ["derive"]
```

Comment:
- This is feeling redundant
- This is less like code someone would run

One idea
- Move the snippet below to be above this
- Consider changing it to a `console` block with `cargo add clap -F derive`

Reply:
Agreed, that's better, it avoids duplication and the information is in a more logical order.

I didn't know about that `cargo add` command, thanks for introducing it to me.

I've amended the commit such that it places the console block above the code example as suggested.